addons/binding/org.openhab.binding.nuki/pom.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.binding</groupId>
+    <artifactId>pom</artifactId>
+    <version>2.3.0-SNAPSHOT</version>
+  </parent>
+  <properties>
+    <bundle.symbolicName>org.openhab.binding.nuki</bundle.symbolicName>
+    <bundle.namespace>org.openhab.binding.nuki</bundle.namespace>
+  </properties>
+  <groupId>org.openhab.binding</groupId>
+  <artifactId>org.openhab.binding.nuki</artifactId>
+  <version>2.3.0-SNAPSHOT</version>
```

Comment:
Please update to 2.4.0-SNAPSHOT

And please also correct the indentation in this file. Should be tabs instead of spaces.

Reply:
yes, fixed this already